packages/expo-cli/src/commands/eject/prebuildAppAsync.ts

```diff
@@ -50,6 +50,13 @@ export async function prebuildAsync(
   const { exp, pkg } = await ensureConfigAsync({ projectRoot, platforms });
   const tempDir = temporary.directory();
 
+  const template = options.template || process.env.EXPO_PREBUILD_TEMPLATE;
+  const skipDependencyUpdate =
+    options.skipDependencyUpdate ||
+    (process.env.EXPO_PREBUILD_SKIP_DEPENDENCY_UPDATE
```

Comment:
Adding two env variables seems a bit verbose, we want to minimize the amount of options/input users have because it increases the maintenance cost of CLI.

Would be better if a user could change the prebuild script or add prebuild flags imo.

Reply:
They mimic the existing parameters though, so we're not really _adding_ options here.

If I understand your alternative approach correctly, you would have something similar to `gradleCommand` in your eas.json, for `prebuildCommand`, and you can pass in all existing parameters that way? While I like that idea, the footprint for that change seems a bit bigger. Within the context of `expo-cli`, there's no concept of eas build config, for starters. Also see @wkozyra95's comment on the issue: https://github.com/expo/eas-cli/issues/887#issuecomment-1005895174